src/pymovements/gaze/gaze_dataframe.py

```diff
@@ -214,6 +218,11 @@ def __init__(
         self.n_components = _infer_n_components(self.frame, column_specifiers)
         self.experiment = experiment
 
+        if events is None:
+            self.events = pm.EventDataFrame()
+        else:
+            self.events = events.copy()
```

Comment:
we should have this structure in the same manner as [`self.frame`](https://github.com/aeye-lab/pymovements/blob/01b8c0389107fa21bccf151ed32e3bff21481263/src/pymovements/gaze/gaze_dataframe.py#L183-L187). imo, the `self.events` way is more readable.


btw, I just noticed that we initialize with data and then rename it to frame. I see where we are coming from but it might lead to user confusion

Reply:
Yes currently using data is a bit confusing, but I also think both terms `frame` leads to similar confusion, because if we pass events, they are data/frame too, right? So what's the difference?

The change I proposed last week also included, that we will slowly migrate into a `Gaze` class, that has `signal` and `events` (and `stimulus` and `meta` and so on) attributes. Let's discuss again and solve the confusion when we start working on that task.